### PR TITLE
Create a zip file suitable for publishing

### PR DESCRIPTION
**Pull request by XVincentX**
This PR will make sure that both zip and crx files gets generated for each build, so we can then automate the publishing process in a near future.

### CODE REVIEW

**Comment by XVincentX** [comment]
[Figure out the Chrome Extension publishing process](https://trello.com/c/3B8CQEO7/288-figure-out-the-chrome-extension-publishing-process)

**Comment by XVincentX** [comment]
@JackuB I added more commits, so you might want to have another look into.

File: extension/manifest.json [hunk 1]
```diff
@@ -1,8 +1,8 @@
 {
   "manifest_version": 2,
+  "name": "Apiary Chrome Extension",
   "key": "MIIBIjANBgkqhkiG9w0BAQEFAAOCAQ8AMIIBCgKCAQEArmcFCbSNikigxEi5/RX1AQ0mEQzMsqlQfikEJdkjjVbDol7UqwKIVioHvn+lWI+o5dyj6qSTXlwB7N9aNXHJFcwI61eU+aWVLB0RObmS5no8URc12GBXn30SzYa89IRSj5hWPnzaVZLlQ0JincaPfBd3fCHVFgAPMOazqv8a/RxnkpwORvhMe5hRDrMznRDCBXxGqqw7qHyflylS8E/qxBl7DkdtMV/YRUpjMUFpvt6JEOE7dLsH+TIUFrp/XCBZo+TPxNkPVxYLaU8bcbwKaQQ2PDL1dwrQSgp0meM4I2ob3jW0v2nkAyw2kwRNXWGTAr7qqzdxSYBycXdWkUP/BwIDAQAB",
-  "name": "Apiary Proxy Chrome Extension",
-  "version": "1.0.0",
+  "version": "0.0.5",
```
Comment (JackuB) [inline extension/manifest.json]: How do you want to keep versioning this? You okay with manual?

File: circle.yml [hunk 2]
```diff
@@ -1,24 +1,31 @@
...
   cache_directories:
     - "client/node_modules"
 
 test:
   pre:
-    - npm run build:extension
     - npm start:
         background: true
+  post:
+    - jq 'del(.key)' ./extension/manifest.json > tmp.$$.json && mv tmp.$$.json ./extension/manifest.json
```
Comment (JackuB) [inline circle.yml]: Could you please add a comment on what's happening on this line?

File: extension/manifest.json (hunk 1 above)
Comment (XVincentX) [inline extension/manifest.json]: No - I really want to switch to semantic versioning and propagate this somehow in this file as well but - not in this PR

File: circle.yml (hunk 2 above)
Comment (XVincentX) [inline circle.yml]: Makes sense.

**Comment by XVincentX** [comment]
@JackuB `react-scripts` was before the acquisition so it should be fine. Unfortunately yes, `crx` might need a verification.

File: extension/manifest.json (hunk 1 above)
Comment (JackuB) [inline extension/manifest.json]: If you want to start using semrel (ask Javor how awesome it is) I recommend you practicing the commit message style :+1:

**Comment by XVincentX** [comment]
I'll keep this as a reference branch

**Comment by XVincentX** [comment]
@JackuB Can you look at this again? I've modified the thing so we prepare just a zip file, no more crx!

**Pull request merged**